internal/app/app.go

```diff
@@ -107,6 +114,7 @@ func (p *PolicyAutomationApp) ClusterReview() error {
 	}
 
 	evalResults := make([]*policy.PolicyEvaluationResult, 0)
+	// FIX
```

Comment:
The below should work fine with a dump as well, I'm not sure to what this "FIX" refers to

Reply:
thanks for spotting this.